### ISSUE TITLE
8gpu rig

### ISSUE THREAD

**Issue by AcharyaOne** [ghostlander/nsgminer#12]
![image](https://user-images.githubusercontent.com/31579891/30058186-c157644e-9231-11e7-9051-37eb9205593b.png)
Using integrated graphics as primary display.
Any idea what the problem is?

**Comment by ghostlander** [comment]
It could be out of video or system memory. Decrease your intensity.

**Comment by AcharyaOne** [comment]
60g gb page file set with 8gb ram, I tried multiple values, it just won't work
nsgminer.exe --api-port 24028 --api-listen --neoscrypt -o stratum+tcp://europe.hub.miningpoolhub.com:20510 -u AcharyaOne.multipoolminer -p x --gpu-threads 1 --worksize 64 --intensity 9 --thread-concurrency 64 --gpu-platform 1

**Comment by ghostlander** [comment]
8Gb RAM isn't much for 8 GPUs. Do you run Windows or Linux?

**Comment by AcharyaOne** [comment]
windows, i know most people run 7 gpu rig with just 4gb ram

**Comment by ghostlander** [comment]
NeoScrypt is more memory intensive than most other algorithms. Windows has been problematic for 4+ GPU rigs for a long time. I suppose you run Windows 10. Not all drivers work well, v16.9.2 to v16.11.5 are reported good for 6+ RX480 GPUs.

**Comment by AcharyaOne** [comment]
Yes it is windows 10 with 8 gtx1070, with latest drivers installed, had some miners that wouldn't work with older version(382.53) but neither nsgminer anyway.
